changelog/1035.deprecate.rst

```diff
@@ -0,0 +1,6 @@
+:attr:`enum.PartyType` is obsolete.
+
+The PartyType enum contains IDs of voice activities, which aren't documented in any official capacity. New activities get added/removed constantly, so keeping up with maintaining that list is rather difficult.
+
+Since the introduction of this enum, activities have been released officially.
+If anything, providing activities should really be more of a bot's responsibility, not the library's, so we should consider deprecating and removing this enum.
```

Comment:
```suggestion
The :class:`PartyType` enum is deprecated, as voice activity IDs are not officially documented and the enum regularly becomes outdated.
```
It doesn't have to be too wordy, and multiline changelogs don't render that well:
![image](https://github.com/DisnakeDev/disnake/assets/8530778/42e222fc-885c-4df1-93a6-9054a702ae42)